### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by whiz25**
This PR delivers the "The Next Web" clone.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            <ul>
+                <li><i class="fab fa-facebook"></i></li>
+                <li><i class="fab fa-twitter"></i></li>
+                <li><i class="fab fa-instagram"></i></li>
+                <li><i class="fab fa-quora"></i></li>
+            </ul>
+        </div>
+    </footer>
+
+</body>
```
Comment (mcrd25) [inline index.html]: HTML file passed all files! Well done! 👍 .
Tab spacing is also consistent.

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+  }
+  
+}
+
+
+
+
+
+
+
```
Comment (mcrd25) [inline assets/css/main.css]: CSS file passed all tests! Well done! 👍 .
Tab spacing is also consistent.

**Comment by grace-mugoiri** [comment]
-fixed the media queries responsiveness 
-fixed the top two images to be of the same size
-added headers for the images for clear view and understanding
-align all right margin pixels to fit the container class

File: assets/css/main.css [hunk 3]
```diff
@@ -1,337 +1,870 @@
...
+.latest-contents h4:hover{
+  cursor: pointer;
+  color: rgb(255, 67, 34);
+}
+.latest-image{
+  height: 60%;
+}
+.latest-image1{
+  background-image: url(../images/latest1.png);
+  background-repeat: no-repeat;
```
Comment (mcrd25) [inline assets/css/main.css]: Hi 👋,

I noticed that the `background-size` and `background-repeat` properties repeat for the 8 image classes, should add it to the `.latest-image` class and remove (except for `background-image` URL of course) from those classes or you can use the `background` shorthand for each to save space, but that would still be unnecessary duplication.
:) 👍

File: assets/css/main.css [hunk 4]
```diff
@@ -1,337 +1,870 @@
...
+    width: 45px;
+    height: 45px;
+    margin-right: 5px;
+  }
+}
+
 
+.latest-funding{
+  height: 300px;
+  display: flex;
```
Comment (mcrd25) [inline assets/css/main.css]: I noticed this appears 8 times in document maybe you can create a flex class. This, of course, is totally up to you and is not required.

Try to do this with similar repeating styles.

File: assets/css/main.css [hunk 5]
```diff
@@ -1,337 +1,870 @@
...
+}
+.funds-lyft-middle{
+  flex: 0 1 33%;
+  border-bottom: 1px solid rgba(255, 67, 34, 0.404);
+  padding: 10px;
+  background-color: rgba(255, 67, 34, 0.027); 
+}
+.funds-lyft-middle h4{
+  font-size: 14px;
+  font-weight: 100px;
```
Comment (mcrd25) [inline assets/css/main.css]: Hi, should be 100, 200, 500, etc. Not a pixel value. :)

**Comment by grace-mugoiri** [comment]
-fixed sticky nav's logo which was out of bounds of the container.
-Made the right cover articles clickable, social icons and footer links clickable.
-put background-size and repeat on latest image class.
-value for font-weight without pixels.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+            <div id="main-logo"></div>
+            <a href="#" class="nav-items">LATEST</a>
+            <a href="#" class="nav-items">HARD FORK</a>
+            <a href="#" class="nav-items">PLUGGED</a>
+            <a href="#" class="nav-items">THE NEXT POLICE</a>
+            <a href="#" class="nav-items">FUNDAMENTALS</a>
+            <a href="#" class="nav-items">IOT LAB</a>
+            <i class="fas fa-search"></i>
+            <div id="menu"><i class="fas fa-bars"></i></div>
+            <div id="bars"><img src="static/images/menu.png" alt=""></div>
```
Comment (mcrd25) [inline index.html]: Does not load in firefox. 
Screen Shot 2019-04-11 at 10 23 59 AM

**Comment by whiz25** [comment]
this was a very challenging project and we learned a lot doing it.

**Pull request merged**